### ISSUE TITLE
Guidance on Composite Resources

### ISSUE THREAD

**Issue by giuliov** [ChrisLGardner/AzureDscPullServerExtension#2]
This extension is extremely interesting, but I have not understood how it should work with Composite Resources.
Should I add configure an internal Repository accessible from the VSTS Agent?

**Comment by ChrisLGardner** [comment]
Yes you should treat them like any other PS module and build and version them on their own and publish them to an internal nuget repo (either in VSTS or wherever your agent can see), I should have put some documentation together for composite resources.

**Comment by alejandromoncayo** [comment]
could some documentation be added to the wiki on regards to how this can be used within the azure devops pipeline portal?

I am following the wiki but getting the following error on the "Compile Azure DSC Mofs"
##[error]The Automation account was not found.
Since it was able to find the automation account on the "Publish Azure DSC Configuration" step and I can verify the configuration was published, I take it is a configuration issue?

also, after publishing to the automation account, how can we apply the DSC within the Azure DevOps pipeline to a virtual machine?
